modules/text2vec-upstage/module.go

```diff
@@ -0,0 +1,151 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package modupstage
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/weaviate/weaviate/modules/text2vec-upstage/ent"
+
+	"github.com/weaviate/weaviate/usecases/modulecomponents/text2vecbase"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/weaviate/weaviate/entities/models"
+	"github.com/weaviate/weaviate/entities/modulecapabilities"
+	"github.com/weaviate/weaviate/entities/moduletools"
+	"github.com/weaviate/weaviate/modules/text2vec-upstage/clients"
+	"github.com/weaviate/weaviate/modules/text2vec-upstage/vectorizer"
+	"github.com/weaviate/weaviate/usecases/modulecomponents/additional"
+)
+
+const Name = "text2vec-upstage"
```

Comment:
this module is not yet a part of Weaviate you should a possibility of activating it  here in [configure_api.go](https://github.com/weaviate/weaviate/blob/main/adapters/handlers/rest/configure_api.go#L858) file, also you can add it to the `defaultVectorizers` [list](https://github.com/weaviate/weaviate/blob/main/adapters/handlers/rest/configure_api.go#L693) as it's a 3rd-party module